<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexEditReplaceWidget</class>
 <widget class="QWidget" name="HexEditReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>58</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolButton" name="closeButton">
     <property name="toolTip">
      <string>Press to close the window</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="findFormatCombo">
     <property name="toolTip">
      <string>Select the data format of the find data field</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="findtextCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="findPrevButton">
     <property name="toolTip">
      <string>Press to find the previous occurrence</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="findNextButton">
     <property name="toolTip">
      <string>Press to find the next occurrence</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Replace:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="replaceFormatCombo">
     <property name="toolTip">
      <string>Select the data format of the replace data field</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="replacetextCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="replaceButton">
     <property name="toolTip">
      <string>Press to replace the selection</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="replaceSearchButton">
     <property name="toolTip">
      <string>Press to replace the selection and search for the next occurence</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="replaceAllButton">
     <property name="toolTip">
      <string>Press to replace all occurrences</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findFormatCombo</tabstop>
  <tabstop>findtextCombo</tabstop>
  <tabstop>replaceFormatCombo</tabstop>
  <tabstop>replacetextCombo</tabstop>
  <tabstop>findPrevButton</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceSearchButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
